package.json

```diff
@@ -4,8 +4,8 @@
   "version": "1.26.4",
   "license": "Apache 2.0",
   "scripts": {
-    "dev": "gatsby develop -H 0.0.0.0",
-    "dev:clean": "yarn clean && yarn develop",
+    "develop": "gatsby develop -H 0.0.0.0",
```

Comment:
I would really really encourage to reconsider and use `dev` instead of `develop` a large community and docs point to use `dev` and `develop` is longer to type.